### DISCUSSION TITLE
Archlinux guest setup mkinitcpio -p linux not working

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#987] [Q&A] [answered]
hey, so when running `mkinitcpio -p linux` on the vm in qemu, i get this
![image](https://github.com/copy/v86/assets/78228404/839cf714-d84f-455a-b6b4-692e3740fdaf)
what do I do?

**Comment by SuperMaxusa** [comment, marked as answer]
`mkinitcpio -p linux` must be executed on installed chroot, like here
https://github.com/copy/v86/blob/8064e19dac7d1b914852ec618a52e38c520c17e8/docs/archlinux.md?plain=1#L201-L205
https://github.com/copy/v86/blob/8064e19dac7d1b914852ec618a52e38c520c17e8/docs/archlinux.md?plain=1#L221-L222
